test/secure_client.js

```diff
@@ -153,5 +153,25 @@ describe('MqttSecureClient', function () {
         done()
       })
     })
+
+    it('should pass the secureContext to the TLS layer', function (done) {
+      var secureContext = { fakeContext: 'fakeValue' }
+      
```

Comment:
remove trailing spaces here as this makes travis build fail (https://travis-ci.org/mqttjs/MQTT.js/jobs/634893622)